FIAT/discontinuous_raviart_thomas.py

```diff
@@ -25,20 +25,15 @@ def __init__(self, ref_el, degree):
 
         # codimension 1 facets
         for i in range(len(t[sd - 1])):
-            pts_cur = ref_el.make_points(sd - 1, i, sd + degree)
-            for j in range(len(pts_cur)):
-                pt_cur = pts_cur[j]
-                f = functional.PointScaledNormalEvaluation(ref_el, i, pt_cur)
-                nodes.append(f)
+            pts_cur = ref_el.make_points(sd - 1, i, sd + degree - 1)
+            nodes.extend(functional.PointScaledNormalEvaluation(ref_el, i, pt)
+                         for pt in pts_cur)
 
         # internal nodes.  Let's just use points at a lattice
-        if degree > 0:
-            cpe = functional.ComponentPointEvaluation
-            pts = ref_el.make_points(sd, 0, degree + sd)
-            for d in range(sd):
-                for i in range(len(pts)):
-                    l_cur = cpe(ref_el, d, (sd,), pts[i])
-                    nodes.append(l_cur)
+        if degree > 1:
```

Comment:
Does this logic get the case of degree==1 correct?

Reply:
I got rid of the Norwegian convention, and made sure that degree was replaced appropritely.

Reply:
So UFL counted (when it had elements) in Norwegian.  And FEEC does.  We are still instantiating the lowest-order space as RT(1) and Ned(1), right?

Reply:
Correct, in that case we always speak Norwegian.